zebra-network/src/protocol/message.rs

```diff
@@ -105,8 +105,9 @@ pub enum Message {
     /// [Bitcoin reference](https://en.bitcoin.it/wiki/Protocol_documentation#reject)
     Reject {
         /// Type of message rejected.
-        // Q: can we just reference the Type, rather than instantiate an instance of the enum type?
-        message: Box<Message>,
+        // It's unclear if this is strictly limited to message command
+        // codes, so leaving it a String.
```

Comment:
From looking at the Zcashd source it seems to just be the message command code as you mention https://github.com/zcash/zcash/blob/master/src/main.cpp#L5447-L5448 so maybe it could even be a `[u8;12]`?

Reply:
it's a little confusing because the protocol documentation says that it's a `var_str` with a length encoding ...

Reply:
Here's the place that Zcashd processes rejects: https://github.com/zcash/zcash/blob/master/src/main.cpp#L6286-L6308

Reply:
It looks like Zcashd doesn't do anything with it except for add it to logs, so we can send an empty string and it's probably fine?